utils/vott_parser.py

```diff
@@ -27,8 +19,18 @@ def __build_tag_from_VottImageTag(image_tag):
         "y2": image_tag.y_max,
         "width": image_tag.image_width,
         "height": image_tag.image_height,
-        "tags": image_tag.classification_names,
-        "location": image_tag.image_location
+        "tags": [image_tag.classification_names],
+        "location": image_tag.image_location,
```

Comment:
@samiyaakhtar is this image location going to be the location on the local disk? Also is "location" necessary? This is an example of json I see produced from VOTT

```
 "4.JPG": [
            {
                "x1": 242.69947363429438,
                "y1": 111.94245383522727,
                "x2": 386.5604846358118,
                "y2": 288.6293767755682,
                "width": 724,
                "height": 483,
                "box": {
                    "x1": 242.69947363429438,
                    "y1": 111.94245383522727,
                    "x2": 386.5604846358118,
                    "y2": 288.6293767755682
                },
                "UID": "9022c82d",
                "id": 3,
                "type": "Rectangle",
                "tags": [
                    "land"
                ],
                "name": 1
            }
        ],
```

Reply:
Location may not be necessary to load it but the other fields are, however when i give it a url location it loads the labels successfully as long as they have the other mandatory fields

Reply:
@samiyaakhtar let's remove it then.

Reply:
removed